geode-core/src/main/java/org/apache/geode/cache/query/internal/CompiledSelect.java

```diff
@@ -941,17 +940,16 @@ private SelectResults prepareEmptyResultSet(ExecutionContext context, boolean ig
     SelectResults results;
     if (this.distinct || !this.count) {
```

Comment:
This could stand to be flattened as well, although it will be more difficult that the above.  I could go either way on this chunk.